extensions/summarize-youtube-video-with-ai/src/summarizeVideo.tsx

```diff
@@ -35,114 +23,57 @@ export type Preferences = {
   anthropicModel: string;
 };
 
-const SummarizeVideo = (
+export default function DeprecationNote(
   props: LaunchProps<{
-    arguments: SummarizeVideoProps;
+    arguments: SummarizeVideoWithProps;
   }>,
-) => {
-  const [summary, setSummary] = useState<string | undefined>();
-  const [summaryIsLoading, setSummaryIsLoading] = useState<boolean>(false);
-  const [transcript, setTranscript] = useState<string | undefined>();
-  const [videoData, setVideoData] = useState<VideoDataTypes>();
-  const { pop } = useNavigation();
-  const { video } = props.arguments;
-
-  if (!ytdl.validateURL(video) && !ytdl.validateID(video)) {
-    showToast({
-      style: Toast.Style.Failure,
-      title: "Invalid URL/ID",
-      message: "The passed URL/ID is invalid, please check your input.",
-    });
-    return null;
-  }
+) {
+  const preferences = getPreferenceValues() as Preferences;
+  const { chosenAi } = preferences;
 
-  useEffect(() => {
-    getVideoData(video)
-      .then(setVideoData)
-      .catch((error) => {
-        showToast({
-          style: Toast.Style.Failure,
-          title: ALERT.title,
-          message: "Error fetching video data: " + error.message,
-        });
+  switch (chosenAi) {
+    case "anthropic":
+      showToast({
+        style: Toast.Style.Failure,
+        title: "Command has changed",
+        message: "Please run `summarizeVideoWithAnthropic` or enter your anthropic API key above",
```

Comment:
Maybe we should show the commands title in all the cases instead of the name?

Reply:
Fair point, I'll adjust 😊